test/test_api.py

```diff
@@ -0,0 +1,5 @@
+"""Tests for ansiblecompat package."""
+
+
+def test_placeholder() -> None:
+    """Placeholder test."""
```

Comment:
I normally also add `assert True` to such placeholder tests. But it's mostly a matter of taste.